src/Core/Resolvers/IQueryBuilder.cs

```diff
@@ -64,6 +64,14 @@ public interface IQueryBuilder
         /// <returns></returns>
         public string BuildStoredProcedureResultDetailsQuery(string databaseObjectName);
 
+        /// <summary>
+        /// Builds the query to determine the number of enabled triggers on a database table.
+        /// Needed only for MsSql.
+        /// </summary>
+        /// <returns></returns>
+        /// <exception cref="NotImplementedException"></exception>
+        string GetQueryToGetEnabledTriggers() => throw new NotImplementedException();
```

Comment:
I agree that you are 'building' a query here, however this query is explicitly related to metadata fetching. In my opinion, this would be better suited for the metadata provider, since there would presumably be database engine specific query text. Also, why add method that isn't used?